lib/ExecutionEngine/ExecutionEngine.cpp

```diff
@@ -162,30 +153,32 @@ void ExecutionEngine::generateIR(CompilationMode mode, Function *F) {
   }
 
   /// Prepare the IR container to handle our function.
-  IR_->setGraph(F);
+  auto IR = llvm::make_unique<IRFunction>(F);
```

Comment:
Is there a reason we use `llvm::make_unique` instead of `std::make_unique`?

Reply:
I thought we'd like to keep c++11 support, but if it's c++14 std:: makes sense.

Reply:
I seem to remember the g++ version we use for coverage on Travis didn't support std::make_unique.  If we can bump that to something newer, though, I'm all in favor of c++14

Reply:
I'm pushing this: https://github.com/pytorch/glow/pull/1251
I think that should resolve the issue you have ATM.

Reply:
not so fast .... :) failing.

Reply:
Issues are resolved now, I'll merge https://github.com/pytorch/glow/pull/1251
After that, you should be able to switch.